vignettes/including-adam-data-in-teal.Rmd

```diff
@@ -1,33 +1,27 @@
 ---
-title: "Input `ADaM` data in a teal application"
+title: "Using ADaM Data in teal Applications"
 author: "NEST CoreDev"
 date: "2022-04-20"
```

Comment:
@chlebowa would you mind removing dates from vignettes?
We had a conversation about dates with @donyunardi in here https://github.com/insightsengineering/teal.logger/pull/55#issuecomment-1699896071 and we decided to remove them. I keep going through packages and I did remove in other places as well. The removal in here would be consistent with other packages. Thanks!